air-quality-backend/system_tests/utils/api_utilities.py

```diff
@@ -38,7 +39,9 @@ def get_list_of_key_values(response_json, key: str) -> list:
     return [forecast[key] for forecast in response_json]
 
 
-def seed_api_test_data(collection_name: str, test_data_list: list[Forecast]):
+def seed_api_test_data(
```

Comment:
We should probably move this into a database_utils file, adding data to the database isn't just for the API, we may also want this for ETL or UI tests at some point. Doesn't need to be this PR.